components/chart/tooltip/overview.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Overview
+page_title: Chart for Blazor | Tooltip Overview
+description: Tooltip for the Blazor Chart
+slug: chart-tooltip-overview
+tags: telerik,blazor,chart,tooltip,tooltips
+published: True
+position: 1
+---
+
+# Tooltip for Telerik Blazor Chart
+
+The Telerik Chart provides a tooltip for its data points. They can be set either to a specific `<ChartSeries>` or as a [shared]({%slug chart-tooltip-shared%}) tooltip for all categories.
+
+In this article:
+
+* [Basics](#basics)
+* [Customization](#customization)
+	* [Parameter Settings](#parameter-settings)
+	* [Template](#template)
+
+
+## Basics
+
+You can enable tooltips for the data points by setting the `Visible` parameter of the `<ChartSeriesTooltip>`, nested inside the `<ChartSeries>` tag, to `true`. By default the value of the point will be presented whey hovered over.
```

Comment:
Consider making this an ordered list to see if it will be clearer to read and easier to follow